### ISSUE TITLE
Room list: Clicking on a room when filtered moves it to the top

### ISSUE THREAD

**Issue by jryans** [element-hq/element-web#14888]
@BillCarsonFr: If you filter for a room and then click on it, it will jump to the top of the sublist, which seems surprising.

See also [this video](https://matrix.to/#/!RbKRuLTvHIhArQcvjr:matrix.org/$1596716855609305qQXyw:matrix.org?via=matrix.org&via=vector.modular.im&via=t2l.io).

**Comment by turt2live** [comment]
This is intended to prevent rooms from disappearing in the list when filtering ends.

**Comment by jryans** [comment]
Right, I assumed it was related to that change. This was reported by @BillCarsonFr in the room, not actually sure what we want to do here per se.

**Comment by turt2live** [comment]
this was developer UX btw as parity for the old list, so might not be the right answer anyways.

An option might be to keep the room as sticky where it is when filtering happens, and when the user stops filtering the sticky position doesn't change (if you click the 2nd result of filtering, then when you stop filtering the room will be 2nd).

This is mostly confusing to people who use stable ordering options, as this breaks those expectations (almost intentionally).

**Comment by t3chguy** [comment]
Fixed by spotlight
